mappings/net/minecraft/client/gui/screen/world/WorldListWidget.mapping

```diff
@@ -2,6 +2,11 @@ CLASS net/minecraft/class_528 net/minecraft/client/gui/screen/world/WorldListWid
 	FIELD field_19132 DATE_FORMAT Ljava/text/DateFormat;
 	FIELD field_19133 UNKNOWN_SERVER_LOCATION Lnet/minecraft/class_2960;
 	FIELD field_19134 WORLD_SELECTION_LOCATION Lnet/minecraft/class_2960;
+	FIELD field_26606 FROM_NEWER_VERSION_ONE_TOOLTIP Lnet/minecraft/class_2561;
```

Comment:
Not sure what version 1 and version 2 refers to here.

Reply:
It's a two line translation key: `selectWorld.tooltip.fromNewerVersion1` and `selectWorld.tooltip.fromNewerVersion2`. It isn't "... version one", it is "... version, line one"

Reply:
```suggestion
	FIELD field_26606 FROM_NEWER_VERSION_FIRST_LINE Lnet/minecraft/class_2561;
```
Something like this is better